### ISSUE TITLE
[IMPROVEMENT] open .vm2 files without renaming to .ps2

### ISSUE THREAD

**Issue by retrosapien** [joack/MyMC-Dual#1]
Im not sure if you still care/maintain this tool, its been a while since last update. ;-)

But if you do;

I noticed the open dialog added the option of .bin files in addition to the .ps2 file the original mymc opens. Could you add the .vm2 file extension to that list? It would save the trouble of renaming them before importing the card to the program.

As far as i know, a .vm2 file is the decrypted form of a .vme from the ps3. It is the exact same file as a .ps2 card, i think.

**Comment by joack** [comment]
I'm not really sure if both files share the same data structure; in any case, you can always rename the .vme extension to .ps2 and test it out. If it works let me know.
------------------
Edit: 
Sorry, I was a bit distracted and didn't read the title correctly.
Yes sure, I can add it.

**Comment by retrosapien** [comment]
You can't rename the vme. It is encrypted.

But the ps3 exports the save as a vm2. Which is just a .ps2 file, afaik. I have re named it and both programs open it fine.

If it requires a big commit, then don't worry about it. I was hoping it would only be an extra line of code.



Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by joack** [comment]
It's ok, I didn't realize that I had made a spelling mistake , you can download the new version here:

https://github.com/joack/MyMC-Dual/releases

Please, let me know if this work as intended

**Comment by retrosapien** [comment]
I installed new .net and tried new version of mymc dual (5.0.1). 
Opening a .vm2 file seams to work as expected. 

TL;DR

There is no other mymc dual dev, right? 
The one i was using had a zip named "MyMC Dual XP v5"
It is different then the one on here, isn't it?
I never had to install ms .net 4.5 when i used it, though. 
It is taking forever to install. 
A quick search returned these two pages;
https://www.psx-place.com/threads/mymc-dual-by-joack.19024/
https://forum.cdrinfo.pl/f106/ps2-mymc-dual-96402/

i'm guessing i used the one from psx-place, as i believe the other site is polish.

it mentions a version for xp, and a version for windows 7 and up, but what about vista?

Anyway, i installed new .net and tried new version of mymc dual (5.0.1). 
Opening a .vm2 file seams to work as expected. 
Opening a .vmc with the all files filter, did not. it crashed the program. (ecc checker)
Only files on real ps2 hardware need ecc, right?
Opening a .bin with the all files filter worked fine, so i think .vmc is just not supported.
Opening a .ps2 with the all files filter worked fine, too.

I would recommend changing your change log description to say "all files" or "no filter" instead of just "."
Maybe a warning to only use without filter if you know what your doing, or something like that.

Now to dig out my ps2, since i finally have all the tools rounded up to transfer saves to the ps3.
Now, If you could integrate PSVExporter into your app, that would make it the ultimate ps2 save tool.
I think there is another mymc fork on here that has done it, but it is in python 3, i think. and it doesn't open .vm2 files.  ;-) ;-)

Thank you for picking this up after such a long break.
=)

**Comment by joack** [comment]
Hi, thanks for the feedback!
Regarding to your questions:

> There is no other mymc dual dev, right?

This was a personal project, so I was the only guy involved in this repo.

> The one i was using had a zip named "MyMC Dual XP v5"
It is different then the one on here, isn't it?

Is the same, but that exe can run in XP too.

> A quick search returned these two pages;
https://www.psx-place.com/threads/mymc-dual-by-joack.19024/
https://forum.cdrinfo.pl/f106/ps2-mymc-dual-96402/
i'm guessing i used the one from psx-place, as i believe the other site is polish.

I belive that polish site grabed the zip file from psx-place and share it in their blog

> it mentions a version for xp, and a version for windows 7 and up, but what about vista?

Vista?! No one use that O.S. anymore. When a was creating this new gui a was using Win 7.

>Only files on real ps2 hardware need ecc, right?

Idk, that question is far from my knowledge.

> Now, If you could integrate PSVExporter into your app, that would make it the ultimate ps2 save tool.

The idea behind of this project was just to enhance the UI of Ross Ridge's tool. I want to keep it as simple as possible, so I can't do that, very sorry.

> Thank you for picking this up after such a long break. =)

Thanks to you too!
